packages/inter-protocol/test/test-clientSupport.js

```diff
@@ -39,26 +46,46 @@ test('Offers.auction.Bid', async t => {
           callPipe: [['makeBidInvitation', [atom.brand]]],
         },
         proposal: {
-          exit: { onDemand: null },
           give: { Currency: ist.make(4_560_000n) },
         },
         offerArgs: {
           offerBidScaling,
-          want: atom.make(1_230_000n),
+          want: { brand: atom.brand, value: 10_000_000_000n },
         },
       },
     );
   });
 
   const price = 7;
   const offerPrice = makeRatio(7n, ist.brand, 1n, atom.brand);
-  t.log({ price, offerPrice });
   t.deepEqual(
-    Offers.auction.Bid(brands, {
+    Offers.auction.Bid(parseAmount, {
       offerId: 'by-price2',
-      wantCollateral: 1.23,
-      giveCurrency: 4.56,
-      collateralBrandKey: 'ATOM',
+      give: '4.56IST',
+      price,
+      desiredBuy: '10_000ATOM',
+    }),
+    {
+      id: 'by-price2',
+      invitationSpec: {
+        source: 'agoricContract',
+        instancePath: ['auctioneer'],
+        callPipe: [['makeBidInvitation', [atom.brand]]],
+      },
+      proposal: { give: { Currency: ist.make(4_560_000n) } },
+      offerArgs: {
+        offerPrice,
+        want: { brand: atom.brand, value: 10_000_000_000n },
+      },
+    },
+  );
+
+  t.deepEqual(
+    Offers.auction.Bid(parseAmount, {
+      offerId: 'by-price2',
+      desiredBuy: '10_000ATOM',
```

Comment:
Can you do failing tests here? If `desiredBuy` is not included, it should be an error.

Reply:
done.